recipes/cpython/all/conandata.yml

```diff
@@ -18,6 +21,16 @@ sources:
     url: "https://www.python.org/ftp/python/3.8.19/Python-3.8.19.tgz"
     sha256: "c7fa55a36e5c7a19ec37d8f90f60a2197548908c9ac8b31e7c0dbffdd470eeac"
 patches:
+  "3.13.0":
+    - patch_file: "patches/3.13/3.13.0-0001-_ctypes-ffi.patch"
+      patch_description: "Support shared libffi"
```

Comment:
I would prefer not workarounding cpython to be able to use shared libffi as this recipe is really fragile. Unless there is some official issue/discussion about this, or is really fixing some known error, I would prefer not patching. Same for other patches.

Reply:
Okay, checking older versions, these patches are ported only, nothing new. Sorry the misunderstanding.

Reply:
All that said, I'm not 100% sure why it's there. Supposedly it's there to support a shared libffi on Windows, but I haven't checked to make sure that it actually fixes it. It's possible that it's just legacy.

Reply:
I did a small amount of testing, and I think this patch hasn't been necessary since 3.9. I want to fix the MSVC/all shared build first to properly test this, but for now it seems harmless to keep consistent with the older versions.